### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,10 +108,10 @@ metadata:
 spec:
   deploymentMode: dev
   runtime: spring-boot
-  version: 2.1.16
+  version: 2.1.16   // runtime version
```
Comment (geoand) [inline README.md]: Do comments work like this in YAML?

Comment (metacosm) [reply README.md]: No, but it's just meant as a call-out in the example… Best to remove it?

Comment (geoand) [reply README.md]: Whatever you think it best. I am only thinking that people might copy-paste it and it won't work

File: README.md [hunk 2]
```diff
@@ -36,74 +36,71 @@ The Halkyon Operator requires `Kubernetes >= 1.13` or `OpenShift >= 3.11`.
...
+Such an application, though simple, will require several Kubernetes resources in order to be deployed on a Kubernetes cluster. 
+Furthermore several development iterations are usually required to make the application production ready.
 
-When, during the analysis phase, the developers will discuss the final picture about the solution to be designed, they
-will certainly adopt this very straightforward convention (aka Fluent DSL) to express the different micro-services, their relations and ultimately
-the services needed.
+The entry point to the application is the `fruit-client-sb` application. It connects to a backend REST endpoint implemented
+by the `fruit-backend-sb` application. This application, in turns, uses the services of a PostgreSQL database.
 
-`(from:componentA).(to:componentB).(to:serviceA)`
```
Comment (cmoulliard) [inline README.md]: I disagree that you remove the DSL language like also the text about how developers, analysts will use it to describe, talk about the application to be developed

Comment (geoand) [reply README.md]: I personally find it confusing so I agree with Chris on removing it

Comment (metacosm) [reply README.md]: I don't see the point of introducing a DSL that's not used anywhere else at this point. If we made use of it, why not but that's not the case and doesn't really add anything to the point we're trying to make, in my opinion.

Comment (cmoulliard) [reply README.md]: ok. I will then accept this PR and add later a footnote about this DSL language

**Pull request merged**